specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/MitreCoverage.json

```diff
@@ -0,0 +1,167 @@
+            "schema": {
+              "$ref": "#/definitions/ProductsMitreCoverageList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "Get": {
+            "$ref": "./examples/MitreCoverage/Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ProductMitreCoverageModel": {
+      "description": "A wrapper to the resource of type T, which contains Arm's first level fields such as id, name and type, with the resource properties",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/1.0/types.json#/definitions/ResourceWithEtag"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Gets or sets nested level of properties which contains the resource content",
+          "type": "object",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ProductMitreCoverageProperties"
+            }
+          ],
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ProductMitreCoverageProperties": {
+      "type": "object",
+      "properties": {
+        "productName": {
+          "description": "Gets Or sets the product name that covers a specific tactic and technique.",
+          "type": "string",
+          "readOnly": true,
+          "example": "MDATP"
+        },
+        "vendorName": {
+          "description": "Gets Or sets the vendor of the Product.",
+          "type": "string",
+          "readOnly": true,
+          "example": "Microsoft"
+        },
+        "coverageSize": {
+          "format": "int32",
+          "description": "Gets Or sets the Coverage Size of the product for a specific tactic and technique.",
+          "type": "integer",
+          "readOnly": true,
+          "example": 2
+        },
+        "technique": {
+          "description": "Gets Or sets MITRE technique code.",
+          "type": "string",
+          "readOnly": true,
+          "example": "T1595"
+        },
+        "tactic": {
+          "description": "Gets Or sets MITRE Tactic.",
+          "type": "string",
+          "readOnly": true,
+          "example": "Reconnaissance"
+        }
+      }
+    },
+    "ProductsMitreCoverageList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets array of arm object wrappers",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ProductMitreCoverageModel"
+          }
+        },
+        "nextLink": {
+          "description": "Gets or sets should be in format of: \"{originalRequestUrl}?$skipToken={opaqueString}\" | null\r\nnextLink value must be a valid Uri.\r\nnextLink value must be null or not returned to mark not support or to finish fetching.",
```

Comment:
>Gets or sets should be in format of: \"{originalRequestUrl}?$skipToken={opaqueString}\"  [](http://example.com/codeflow?start=26&length=88)

nextLink is an opaque Uri string. Modifying it isn't supported, so the format doesn't matter.